src/domain.rs

```diff
@@ -502,6 +507,11 @@ impl<F> Domain<F> {
         .expect("system time is too far into the future")
     }
 
+    #[cfg(not(feature = "std"))]
+    fn now() -> u64 {
```

Comment:
This seems unfortunate. Let's instead use `#[cfg]` on the code that _calls_ `now` so that it doesn't need to exist.